tools/asm_processor/asm_processor.py

```diff
@@ -2,7 +2,6 @@
 import argparse
 import tempfile
 import struct
-import copy
 import sys
```

Comment:
this tool is also external https://github.com/simonlindholm/asm-processor
(I'm making changes for the PR author, but just commenting for documentation purposes, and incase the author feels like making PRs upstream)